R/srp_model.R

```diff
@@ -71,8 +71,9 @@ create_srp_model <- function(
 #' @importFrom checkmate assert_vector
 is_valid.srp_model <- function(mdl) { # nolint
   with(mdl, {
-    stopifnot(logodds_mean[1] < logodds_max[1])
-    stopifnot(logodds_mean[2] < logodds_max[2])
+    checkmate::assertTRUE(logodds_mean[length(attr(mdl, "group_id"))] <
```

Comment:
hm, not sure this is correct. why not `all(logodds_mean < logodds_max)`  ?

Reply:
I tried that. But it always gives error if there are more than 1 groups.
It considers only 1 logical variable as input i think

Reply:
can solve it with sum

Reply:
but one another test case fails if I use this method.